CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## Target branch
+
+Pull requests should target the `main` branch. If the change is relevant for a hotfix release, make a note in the PR description so it can be cherry picked after merge.
+
+## Adding to CHANGELOG
+
+All changes focused on users of the released builds should be documented by writing a new entry to [CHANGELOG.md](CHANGELOG.md), under the unreleased version at the top.
+
+Example of changes that should have changelog:
+- Any database change for a stable game
+- Any GUI change (that isn't hidden by `preview mode` or belongs to an experimental game).
+- Changes to multiworld, presets, generator or solver that causes visible changes in the result
+
+Example of changes that should _not_ have a changelog:
+- Changes to experimental games
+- Anything only visible when running from source
+- Code refactoring
+- Dependency changes (except for patcher dependencies, since these can change the results!)
+
+## How to write a good Changelog Entry
+
+- Use full name of areas
+- Mention all new tricks/difficulty pair and in which areas.
+- Someone reading the Changelog should be able to identify which areas a preset they have had changes without having to open the Data Visualizer, but will need to for the precise details.
```

Comment:
It should be discernable which areas are effected when reading the changelog alone. It is acceptable if users need to open the Data Visualizer to understand the exact details of what changed in those areas, however.